src/dlgprefreplaygain.cpp

```diff
@@ -108,8 +108,7 @@ void DlgPrefReplayGain::slotUpdateReplayGainBoost() {
 
 void DlgPrefReplayGain::setLabelCurrentReplayGainBoost(int value) {
     LabelCurrentReplayGainBoost->setText(
-            QString(tr("%1 dB (average %2 dB)")).arg(
-                  QString().sprintf("%+d", value), QString::number(value - 14)));
+            QString(tr("Adjust by %1 dB")).arg(QString().sprintf("%+d", value)));
```

Comment:
by your own list of random replaygain values, an adjustment of -6 will not necessarily result in a -14db target.  Therefore it is not accurate to put the -14db value in the slider text.